app/views/public_activity/webhook/_create.csv.slim

```diff
@@ -0,0 +1 @@
+= CSV.generate_line(['webhook', activity.trackable.url, 'create', '-', activity.owner.username, activity.created_at, "owned by namespace #{activity.trackable.namespace.name}"])
```

Comment:
This is totally unrelated with this PR, however I just realized one thing.

Suppose:

  1. user triggers webhook for namespace "foo"
  2. user downloads CSV audit file
  3. namespace "foo" is renamed "bar"
  4. user download CSV 

The audit entry for event number 1 is reported in two different ways depending from the generation time. I think this is really wrong, the past events of the audit log should always stay the same!

What do you think about that, this is probably worth a dedicated issue.